Cargo.toml

```diff
@@ -34,8 +51,11 @@ p256-cortex-m4 = { version = "0.1.0-alpha.6", features = ["prehash", "sec1-signa
 littlefs2 = { git = "https://github.com/trussed-dev/littlefs2.git", rev = "ebd27e49ca321089d01d8c9b169c4aeb58ceeeca" }
 trussed = { git = "https://github.com/Nitrokey/trussed.git", tag = "v0.1.0-nitrokey.18" }
 trussed-auth = { git = "https://github.com/Nitrokey/trussed-auth", rev = "49c13eae6d9a225676191d4776d514848e4eab5b" }
+trussed-manage = { git = "https://github.com/Nitrokey/trussed-staging.git", branch = "extensions" }
 trussed-rsa-alloc = { git = "https://github.com/Nitrokey/trussed-rsa-backend.git", rev = "2088e2f8a8d706276c1559717b4c6b6d4f270253" }
-trussed-staging = { git = "https://github.com/nitrokey/trussed-staging.git", rev = "59dda984e42fbbbd9b469c773af08b497b913469" }
+trussed-wrap-key-to-file = { git = "https://github.com/Nitrokey/trussed-staging.git", branch = "extensions" }
+
+trussed-se050-manage = { path = "extensions/se050-manage" }
```

Comment:
This should not be a patch. It should be directly in the `trussed-se050-manage` dependency definition. When built locally it will use the `path`, but when published on crates.io it will use the dependency on the registry, following the `version` key.

Reply:
I agree when we are talking about crates published to crates.io.  But that is not possible in the moment and won’t be in the near future.  Until then, we will use Git dependencies, and I don’t know what happens if a Git dependency has a path dependency.  So I chose to keep things simple and consistent.  If you have more information on how Git dependencies with path dependencies work, I’m happy to change it.